### PR TITLE
Create dylan.json

### PR DESCRIPTION
**Pull request by dylanjamesdev**

### CODE REVIEW

File: domains/dylan.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "This will soon be a dev portfolio for myself :)",
+  "repo": "https://github.com/dylanjamesdev",
+  "owner": {
+    "username": "dylanjamesdev",
+    "email": "dylan@tritan.gg"
+  },
+   "record": {
+    "URL": "https://gitlab.com/team-tritan
```
Comment (mtgsquad) [inline domains/dylan.json:9]: Hey, here you need to replace the record with: `"URL": "https://gitlab.com/team-tritan"`, the ending quotation marks are missing.

**Review by mtgsquad — CHANGES_REQUESTED** [review]

File: domains/dylan.json (hunk 1 above)
Comment (dylanjamesdev) [reply domains/dylan.json:9]: Ahh my bad

Comment (dylanjamesdev) [reply domains/dylan.json]: Fixed

**Review by hackermondev — APPROVED** [review]

**Review by phenax — APPROVED** [review]

**Comment by phenax** [comment]
The changes have been published!! It should reflect in less than 24 hours.

## Here's what you need to do next
If your domain points to a server you own, add \`domain-name.is-a.dev\` to your server config. For https, you will have to configure ssl certificate to allow the new subdomain.

### For github pages users,
* Go to your github page repo (\`user/user.github.io\`)
* Open up the **settings** tab
* Scroll down to the **Github pages** section
* In the **Custom domain** text input, enter the domain you registered (\`domain-name.is-a.dev\`)
* Check the **Enforce HTTPS** checkbox below the input
* Give it some time to reflect and you should be good to go


## Need help with your domain?
If you are having trouble setting up your domain, [create an issue](https://github.com/is-a-dev/register/issues/new/choose). I will try my best to get back to you asap!


## Made a mistake in the record?
Don't worry, you can create a new pull request with the corrections


## Love/Hate the service?
**Love it?** Leave it a **star**! Also consider donating so that I can keep this service running forever!
**Hate it?** Please leave your feedback by [creating an issue](https://github.com/is-a-dev/register/issues/new/choose). I would really like to keep improving this service for other users.


## Wanna support this project?
Help me in my mission to keep this service alive forever by donating!

<a href="https://www.buymeacoffee.com/phenax" target="_blank">Buy Me A Coffee</a> <a href="https://liberapay.com/phenax" target="_blank">Liberapay recurring donation button</a>

**Pull request merged**